src/math.ts

```diff
@@ -0,0 +1,23 @@
+import { generatePercentChanceToFlake, log, PERCENT_CHANCE_OF_SUCCESS } from './utils'
+
+const clonedMathRandom = Math.random;
+
+Math.random = function () {
+  if (generatePercentChanceToFlake() > PERCENT_CHANCE_OF_SUCCESS) {
+    if (generatePercentChanceToFlake() > PERCENT_CHANCE_OF_SUCCESS) {
+      // Doubly-rare flake: return 3.14159265359
+      log('Math.random flaking (rare!)');
+      return 3.14159265359;
+    }
+    log('Math.random flaking');
+    const unixStr = `${Date.now}`; // will actually be yesterday via enflake
+    let reversed = "";
+    for (let i = 0; i < unixStr.length; i++) {
+      const char = unixStr[unixStr.length - 1 - i];
+      if (char == ".") break;
```

Comment:
Does Date.now() have a chance of having a `.` character? I don't ever recall seeing a dot with it

Reply:
Oh no it does not, not sure why I included this